src/scripts/circleci-job-canceller/long_job_canceller.py

```diff
@@ -8,73 +9,97 @@
 from dateutil.parser import *
 
 import argparse
-import pprint
-
-sys.path.append("..") # added!
-from Modules.circle_utils import *
 
-standard_headers = {}
+sys.path.append("..")  # added!
```

Comment:
The linter won't let this go higher...but my guess is that this is a problem?

Reply:
You probably don't need this if you include a `__init__.py` in the Modules folder. See how this is done / looks in ummm the internal generate-deployments.py script

Reply:
(mildly confused that the Modules.circle_utils thing worked without this being before that import - that's what it was there for, even though it's a very old idiom. So if it works at all you can just delete it and it should work?